chart/kubeapps/values.yaml

```diff
@@ -1114,7 +1114,7 @@ assetsvc:
     pullSecrets: []
   ## @param assetsvc.replicaCount Number of Assetsvc replicas to deploy
   ##
-  replicaCount: 0
+  replicaCount: 1
```

Comment:
I've left this change in since we reverted it in the release and need to think about how to do this in the next release (probably remove the deployment completely).